CCUI.DAPPI/src/app/state/content/content.effects.ts

```diff
@@ -38,20 +45,52 @@ export class ContentEffects {
     )
   );
 
+  loadRelatedItems$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(ContentActions.loadRelatedItems),
+      mergeMap((action) => {
+        const endpoint = `http://localhost:5101/api/${action.selectedType
```

Comment:
It is a good approach to keep base API urls in a separate file, to further construct the path as needed. So, if you had one defined like `BASE_API_URL=http://localhost:5101`, then your endpoint would look like this:

```suggestion
        const endpoint = `BASE_API_URL/api/models/api/${action.selectedType
```